src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaTimelinePersistenceAdapterTest.kt

```diff
@@ -28,6 +27,7 @@ import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.aVal
 @ExtendWith(MockitoExtension::class)
 class JpaTimelinePersistenceAdapterTest {
```

Comment:
This class has simply been reverted back to its previous state, before I tested the connection with the prison-api.